docs/music.rst

```diff
@@ -3,13 +3,15 @@ Music
 
 .. py:module:: music
 
-This is the ``music`` module. You can use it to play simple tunes, provided
-that you connect a speaker to your board. By default the ``music`` module
-expects the speaker to be connected via pin 0:
+This is the ``music`` module and you can use it to create and play melodies.
+By default sound output will be via the built-in speaker **V2** and via the 
```

Comment:
As an earlier comment, maybe we should mention pin0 first as that's the common mode, and then indicate in v2 is also plays via speaker.

Reply:
Done in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5